app/views/admin/agents/_agent.html.slim

```diff
@@ -4,8 +4,21 @@
 tr id="agent_#{agent.id}"
   td
     - if agent_policy.edit?
-      = link_to agent.reverse_full_name, edit_admin_organisation_agent_path(current_organisation, agent), class: "mr-2"
-      = inactive_tag(agent)
+      - if agent.full_name.present?
+        span= link_to agent.reverse_full_name, edit_admin_organisation_agent_path(current_organisation, agent), class: "mr-2"
+      - unless agent.is_an_intervenant?
+        / les intervenants ne peuvent pas se connecter, il est inutile de les afficher comme "Inactif".
+        - if agent.last_sign_in_at.nil? && agent.invitation_sent_at.present? && agent.invitation_accepted_at.nil?
+          - if agent.invitation_due_at.past?
+            .badge.badge-warning Invitation expirée
+          - else
+            .badge.badge-warning Invitation en attente
+        - elsif agent.last_sign_in_at.nil?
+          / ce cas est très rare et n’arrive plus depuis 2022 car les agents sont connectés tout de suite après avoir accepté l’invitation
+          .badge.badge-warning Jamais connecté·e
+        - elsif agent.last_sign_in_at <= 1.month.ago
+          .badge.badge-warning Dernière connexion > 1 mois
```

Comment:
Je sais pas si ce texte est beaucoup plus explicite que "inactif", peut-être que "Pas connecté·e depuis plus d'un mois" c'est plus clair ?

Reply:
je n’ai pas d’avis très fort sur les deux versions je les trouve assez équivalentes mais vu que ça avait déjà été itéré avec Nesserine et Téo je vais garder la version actuelle « Dernière connexion > 1 mois » si ça te va. Je trouve ça nettement plus explicite que « inactif » perso !

Reply:
on s'est peut-être pas compris, je disais que "Pas connecté·e depuis plus d'un mois" c'est plus clair que "Dernière connexion > 1 mois", en restant équivalent comme sens (je sais que c'est le texte proposé par téo dans un des commentaires, mais je crois pas qu'elle voulait que ça soit au signe de ponctuation près)